tasks/prepare-fbc-release/tests/mocks.sh

```diff
@@ -0,0 +1,20 @@
+# mocks to be injected into task step scripts
+
+#!/bin/sh -ex
+
+function skopeo() {
+  echo "Mock skopeo called with: $*" >> $(workspaces.data.path)/mock_skopeo_log.txt
```

Comment:
One suggestion - something that I started using recently. Normally we would just print this, but here we can't because the output is directly used. But you can still print to stderr like this:
```sh
echo "Mock skopeo called with: $*" >&2
```
That way it will still be visible in the TaskRun logs, but it won't interfere with the workflow.

Reply:
Great suggestion, thanks :)